### PR TITLE
Add postal_code validation

### PR DESCRIPTION
**Pull request by girardinsamuel**
Implements #37

### CODE REVIEW

File: src/masonite/validation/Validator.py [hunk 1]
```diff
@@ -791,6 +791,38 @@ def negated_message(self, attribute):
...
 
+class postal_code(BaseValidation):
+    def __init__(self, validations, locale, messages={}, raises={}):
+        super().__init__(validations, messages=messages, raises=raises)
+        from .resources.postal_codes import PATTERNS
+
+        try:
+            pattern_dict = PATTERNS.get(locale)
+            if pattern_dict["pattern"] is None:
+                # TODO: how to raise ?
```
Comment (josephmancuso) [inline src/masonite/validation/Validator.py]: are these TODOS important for the completion of the feature?

File: src/masonite/validation/Validator.py [hunk 2]
```diff
@@ -791,6 +791,38 @@ def negated_message(self, attribute):
...
+                print("Postal code validation not implemented for {}".format(locale))
+            else:
+                self.pattern = pattern_dict["pattern"]
+                self.pattern_example = pattern_dict["example"]
+        except KeyError:
+
+            # TODO: how to raise an Error in init of a validation rule ?
```
Comment (josephmancuso) [inline src/masonite/validation/Validator.py]: What is this TODO referring to? why can't we raise an error like you are doing? I don't see why we couldn't just raise an error here?

**Review by josephmancuso — CHANGES_REQUESTED** [review]
Looks like a good start, would need some tests here.

File: src/masonite/validation/Validator.py (hunk 2 above)
Comment (girardinsamuel) [reply src/masonite/validation/Validator.py]: I don't know if raising an exception was the good way to handle error in this validation packages. I did not check yet if there were some functions to handle that inside the validation package.
But I am okay with raising an error 👍

**Comment by girardinsamuel** [comment]
> Looks like a good start, would need some tests here.

Yes, I am planning to add some tests and complete this soon

**Comment by girardinsamuel** [comment]
Looks like this PR is ready for review :) 
I am just not sure that every regex patterns is up to date or correct, there are around 181 patterns...let the open source community help us ! I am going to finish the doc update for those new rules (postal_code and files)

**Review by josephmancuso — APPROVED** [review]
I love it

**Pull request merged**